AssistantV1.playground/Pages/Examples.xcplaygroundpage/Contents.swift

```diff
@@ -96,12 +75,4 @@ assistant.deleteExample(workspaceID: workspaceID, intent: intent, text: "Good af
 
 // Cleanup
```

Comment:
If the `// Setup` comment is deleted, then let's delete `// Cleanup` too.

Reply:
The `// Cleanup` is there to signal to the `make_swift_examples` script to exclude that from the example -- it doesn't check for that currently, but we need some indicator.

Reply:
Makes sense. Is the `// Setup` comment not needed for the same reason?

Reply:
Everything up to the first `//:###` is discarded.

But there's no harm in keeping the `// Setup`. I can add it back if you prefer the symmetry.

Reply:
Oh, ok. I think we should add it back.

Reply:
Done.